From: Tommy Webb <tommy@calyxinstitute.org>
Date: Wed, 6 Sep 2023 12:33:49 +0000
Subject: fixup! Enable Certificate Transparency

No guard for the feature because the feature flag is removed.
---
 chrome/browser/about_flags.cc | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -9424,13 +9424,6 @@ const FeatureEntry kFeatureEntries[] = {
      flag_descriptions::kWebMidiDescription, kOsAll,
      FEATURE_VALUE_TYPE(features::kWebMidi)},
 
-#if BUILDFLAG(IS_ANDROID)
-    {"certificate-transparency-enabled",
-     flag_descriptions::kCTEnabledName,
-     flag_descriptions::kCTEnabledDescription, kOsAndroid,
-     FEATURE_VALUE_TYPE(features::kCertificateTransparencyAndroid)},
-#endif
-
 #if BUILDFLAG(IS_CHROMEOS_ASH)
     {"dns-over-https-with-identifiers-reuse-old-policy",
      flag_descriptions::kDnsOverHttpsWithIdentifiersReuseOldPolicyName,
-- 

